src/utils/constant.js

```diff
@@ -0,0 +1,5 @@
+const protocol = window.location.protocol; // 'http:' or 'https:'
+export const backendUrl =
+  protocol === 'https:'
+    ? 'https://backend-service-rojjrgeqna-ue.a.run.app'
```

Comment:
must be replaced with the backend url from environment file.

Reply:
My bad. I will add.